### DISCUSSION TITLE
WalrusLinks

### DISCUSSION THREAD

**Discussion by sycute** [creatorsand/.github#123] [Ideas]
一句话 IDEA：

去中心化的网站收藏平台，通过Walurs技术实现用户间资源共享
repo：https://github.com/sycute/walrus_links

我的角色：

合约开发

社群昵称：

地球观察员

个人简介：

web2区块链开发，web3学习中

产品是给谁用的？

需要收藏/记录 walrus sites 的用户

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
